constructors/service.go

```diff
@@ -56,10 +57,16 @@ func (deps webServiceDependencies) pingService(ctx context.Context, service serv
 	}
 	return
 }
+
 func (deps webServiceDependencies) getGRPCAddress(ports []server.ListenInfo) string {
 	for _, info := range ports {
 		if info.Type == server.GRPCServer {
-			return info.Address
+			switch info.Network {
```

Comment:
Why do you need to add a schema to Address ? It's not an URI

Reply:
It used by grpc dialer when we change the grpc server with unix socket in the callbacks in the PR.

we can get more detail in https://github.com/grpc/grpc/blob/master/doc/naming.md